nodejs/week1/meals/src/backend/routes/meals.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const app = express();
```

Comment:
Remove. Same explanation than the previous file. Also export only router function.